src/Lucid/Model/index.js

```diff
@@ -817,6 +817,12 @@ class Model extends BaseModel {
      */
     if (trx) {
       query.transacting(trx)
+
+      // Override `rollback` method
+      // in order to `unfreeze` the model after the transaction
+      const rollback = trx.rollback
```

Comment:
I believe this will break, if I pass the same `trx` to multiple delete calls

Reply:
Yes, I'm not really happy with this (cf my comment https://github.com/adonisjs/adonis-lucid/pull/373#issuecomment-419025714) do you have any other idea?